src/amo/components/AddonsCard/styles.scss

```diff
@@ -8,12 +8,16 @@
 
     .Card-contents {
       background: $white;
-      border-bottom-left-radius: $border-radius-default;
```

Comment:
Are you sure it's safe to do this? This component is used a lot:

```
$ crg '<AddonsCard' src/
src/amo/components/AddonRecommendations/index.js
166:      <AddonsCard

src/amo/components/AddonsByAuthorsCard/index.js
321:      <AddonsCard

src/amo/components/Collection/index.js
423:            <AddonsCard

src/amo/components/LandingAddonsCard/index.js
53:      <AddonsCard

src/amo/components/SearchResults/index.js
61:        <AddonsCard
```

Reply:
Well, I thought that would be ok but after a closer look, that was a bit silly. I have a better change for what I wanted to achieve here.